### ISSUE TITLE
A11y_AzureDataStudioSQLMigration_MigratetoAzureSQL_MigrationAssessment_Keyboard:There is no visible indication of focus on the OK button when navigating using tab key.

### ISSUE THREAD

**Issue by Shivamd1** [microsoft/azuredatastudio#15763]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#SH_AZUREDATASTUDIOSQLMigration_June2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#WCAG2.4.7;#AINotInScope;#Keyboard;#Focusvisible

### Environment Details:
Application Name: SQL Azure Data Studio
Windows Version: Win10

### Repro Steps:

1. Launch the application-Azure Data Studio-Insiders and install extension Azure SQL Migration.
2. Connect to any sql server.
3. Tab till Azure SQL migration Tab and press enter.
4. Tab till Migrate to azure sql and press enter.
5. Add azure account and tab till next button and press enter.
6. Select SQL server on azure virtual machine radio button and tab till view/select button and press enter.
7. Tab till Ok button and check if the focus is visible on OK button.

### Actual:
There is no visible indication of focus on the OK button.
**Similar issue is observed throughout the application where OK button,Close button and Next button is present.**

### Expected:
The OK button should have a proper visible indication of focus.

### User Impact:
Keyboard only users will not be able to see where the focus is situated at.

**Comment by aasimkhan30** [comment]
Depends on the fix for https://github.com/microsoft/azuredatastudio/issues/15740

**Comment by aasimkhan30** [comment]
Cannot repro this issue in the main. Closing this for now

**Comment by prabhudevu** [comment]
I have verified the issue in the below build

Version: 1.43.0-insider (user setup)
Commit: https://github.com/microsoft/azuredatastudio/commit/afafee844ce2b07eb55bdb6f40ee1ce88c0bbf4e
Date: 2023-03-29T05:28:54.263Z
VS Code: 1.67.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.22621

**GithubTags:**#Regressed:03-29-23;

@aasimkhan30  I could not find "Ok" and "close" button but "Select", "Cancel". "Previous" buttons are there and keyboard focus is visible.

Had "OK" and "Close" buttons deprecated and UI changed?

![15763](https://user-images.githubusercontent.com/98821792/228527207-b0c96874-7331-4fee-a394-90b611040a90.gif)

**Comment by aasimkhan30** [comment]
Yes, the okay button is renamed to next button and close button is renamed to cancel

**Comment by prabhudevu** [comment]
@aasimkhan30 Thanks for the confirmation. closing the bug as per above comment 
 
 **GithubTags**#Closed;#Regressed:03-30-23;
